packages/typescript-vfs/src/index.ts

```diff
@@ -641,9 +641,9 @@ export function createVirtualLanguageServiceHost(
 }
 
 const requirePath = () => {
-  return require(String.fromCharCode(112, 97, 116, 104)) as typeof import("path")
+  return require('path') as typeof import("path")
```

Comment:
@orta thanks for your reply, can it at least be refactored to ?
```suggestion
  return typeof process === 'undefined' ? require('path') : require(String.fromCharCode(112, 97, 116, 104)) as typeof import("path")
```